clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/PartitionConsumptionState.java

```diff
@@ -269,6 +269,11 @@ enum LatchStatus {
 
   private final Schema keySchema;
   private KeyUrnCompressor keyUrnCompressor;
+  /**
+   * Indicates whether a bootstrapping current version replica has switched to high priority current version pool.
+   * In a PCS's lifetime, this will only be flipped at most once.
+   */
+  private boolean hasSwitchedConsumptionPool;
```

Comment:
Can we changed the name to avoid mention consumption pool, maybe hasBoostrapedAsCurrentVersion or other?